.agents/skills/tinyengine-dsl-generator/scripts/check_css.py

```
@@ -0,0 +1,249 @@
+        try:
+            # 解析CSS规则列表
+            rules = tinycss2.parse_stylesheet(css, skip_comments=False)
+
+            # tinycss2会抛出解析错误
+            for rule in rules:
+                if rule.type == 'error':
+                    self.errors.append(f"Parse error: {rule.message}")
+
+            return len(self.errors) == 0
+
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)